code_snippets/api-metric-metadata-get.py

```diff
@@ -0,0 +1,12 @@
+from datadog import initialize, api
+
+options = {
+    'api_key': '9775a026f1ca7d1c6c5af9d94d9595a4',
+    'app_key': '87ce4a24b5553d2e482ea8a8500e71b8ad4554ff'
+}
+
+initialize(**options)
+
+metric = 'system.cpu.idle'
+
+print api.Metadata.get(metric_name=metric)
```

Comment:
Also this api call seems to return a 403  when settings API keys.  My concern more than the 403 is that the error returned contains a bunch of html which seems odd for an API


Compiling site…
Traceback (most recent call last):
  File "/var/folders/cj/7zqrm9kj3cq896q91m3b42100000gn/T/foo20170215-68252-84dhlh", line 16, in <module>
    pprint(wrapper())
  File "/var/folders/cj/7zqrm9kj3cq896q91m3b42100000gn/T/foo20170215-68252-84dhlh", line 13, in wrapper
    return api.Metadata.get(metric_name=metric)
  File "/usr/local/lib/python2.7/site-packages/datadog/api/metadata.py", line 24, in get
    return super(Metadata, cls).get(metric_name)
  File "/usr/local/lib/python2.7/site-packages/datadog/api/resources.py", line 130, in get
    return APIClient.submit('GET', cls._class_url + "/" + str(id), **params)
  File "/usr/local/lib/python2.7/site-packages/datadog/api/api_client.py", line 147, in submit
    raise ValueError('Invalid JSON response: {0}'.format(content))
ValueError: Invalid JSON response: 
<!DOCTYPE html>